src/qt/textctrl.cpp

```diff
@@ -161,15 +163,26 @@ int wxTextCtrl::GetNumberOfLines() const
 
 bool wxTextCtrl::IsModified() const
 {
-    return false;
+    if( !IsMultiLine() )
```

Comment:
Why use negative test here? I'd either write this as just `if ( IsMultiLine() )` and exchange the branches or, if you really want the single-line version to come first, just use `if ( IsSingleLine() )` instead.

Also, this is minor, but please put a space between `if` and `(`.

Both remarks apply to the other changes too, of course.

Reply:
@vadz ,
This file contains both negative and positive if checks. However, searching reveals that negative check prevails.
Therefore I used a negative checks here.

If you think that the code should be changed, I can change the condition throughout the file for consistency.

Let me know if you prefer to do it this way.

TIA!

Reply:
I think it's better to write the checks in more straightforward way, but I'm not sure if it's worth changing the existing code to avoid them. So I'd just write the new tests in the more natural and readable way but I wouldn't change anything else.

Reply:
@vadz ,
Got it.
I will update the code tonight.